cmake/modules/SwiftSupport.cmake

```diff
@@ -0,0 +1,37 @@
+
+# Returns the current achitecture name in a variable
+#
+# Usage:
+#   get_swift_host_arch(result_var_name)
+#
+# If the current architecture is supported by Swift, sets ${result_var_name}
```

Comment:
What actually reads `result_var_name`? I don't see it referenced anywhere else in these files. It'd be good to document this.

Reply:
Its the output variable, not an input variable.

Reply:
I understand that, but what reads this output?

Reply:
The caller of the function will read the output (which can be anything really, its completely opaque to the function).